tutorcredentials/templates/credentials/build/credentials/Dockerfile

```diff
@@ -102,15 +104,15 @@ FROM minimal AS production
 # Install system requirements
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked apt update \
-    && apt install -y libxml2 libmysqlclient-dev mime-support
+    && apt install -y libxml2 libmysqlclient-dev media-types mailcap
 
 # From then on, run as unprivileged "app" user
-ARG APP_USER_ID=1000
+ARG APP_USER_ID=1001
```

Comment:
This is almost certainly not a good idea. See this comment for more information: https://github.com/overhangio/tutor/pull/1117/files#r1795117242